docker/nightly/centos7/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM centos:latest
+
+LABEL maintainer="Jumping Yang <yzpcreate@qq.com>"
+LABEL readme.md="https://github.com/Microsoft/omi/blob/master/docker/README.md"
+LABEL description="This Dockerfile will install the latest code of OMI."
+
+ARG fork=Microsoft
+ARG branch=master
```

Comment:
I'm thinking the stable branch would be better than the master branch. Master branch is totally untested code.

Unless, of course, this is needed for automation/testing purposes, in which case this probably doesn't belong in this repo (perhaps the automation repo)?